api/diagnostics.go

```diff
@@ -249,12 +238,17 @@ func (j *DiagnosticsJob) flushReport(zipWriter *zip.Writer, fileName string, rep
 	}
 }
 
-func (j *DiagnosticsJob) collectDataFromNodes(nodes []dcos.Node, summaryReport *bytes.Buffer,
-	summaryErrorsReport *bytes.Buffer, zipWriter *zip.Writer) {
-	j.setJobProgressPercentage(0)
```

Comment:
Did this get lost in the process? Nothing sets `0` anymore?

Reply:
It is set here https://github.com/dcos/dcos-diagnostics/pull/90/files#diff-0ee9de7b9d9927cce9fb53251662d72aR167 I moved it because I thought it will be clear that we set 0 at the beginning and 100 at the end